example/deployment/AWS-EC2-S3/bash-scripts/runFormattedEC2Example.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+. "$DIR/setScriptPath.sh"
+
+export PALISADE_REST_CONFIG_PATH="/home/ec2-user/example/example-model/src/main/resources/configRest.json"
+
+# Runs the demo example, start to finish, all users, all purposes 
+# java -cp /home/ec2-user/example/example-model/target/example-model-*-shaded.jar uk.gov.gchq.palisade.example.runner.RestExample $1 | /home/ec2-user/example/deployment/bash-scripts/formatOutput.sh
+
+# Runs the example and allows user to pass in a name, file and purpose
+java -cp /home/ec2-user/example/example-model/target/example-model-*-shaded.jar uk.gov.gchq.palisade.example.client.ExampleSimpleClient Alice $1 SALARY | /home/ec2-user/example/deployment/bash-scripts/formatOutput.sh
```

Comment:
comment doesn't reflect the code, we should add input checking to ensure the three arguments are passed in then make sure this line uses them.
Am happy for the commented out line above to stay in this file as it is a useful alternative that could be used

Reply:
The first method in the jar checks to see if 3 args are passed in, if not it will throw an error. Is this still needed?